pageserver/src/tenant/timeline.rs

```diff
@@ -2693,25 +2767,47 @@ impl Timeline {
         Ok(layer_paths_to_upload)
     }
 }
+
 #[derive(Default)]
 struct CompactLevel0Phase1Result {
     new_layers: Vec<DeltaLayer>,
     deltas_to_compact: Vec<Arc<dyn PersistentLayer>>,
 }
 
+#[derive(Debug, thiserror::Error)]
+enum Phase1Error {
```

Comment:
I don't think the phase1error carries it's weight, combining this into the CompactionError.

Reply:
Only one remains: 1560f0ed